src/lib/transaction_snark_scan_state/jbuild

```diff
@@ -1,12 +1,12 @@
 (jbuild_version 1)
 
 (library
- ((name parallel_scan)
-  (public_name parallel_scan)
+ ((name transaction_snark_scan_state)
```

Comment:
I think there should still be a distinct `parallel_scan` library that doesn't depend on any coda specific stuff (like coda-base, snarks etc)

Can you make `transaction_snark_scan_state` a new separate library that depends on parallel-scan?